sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -279,7 +280,15 @@ private JobManifestTasks getJobManifestTasks(TextAnalyticsActions actions) {
                         final ExtractiveSummarizationTask extractiveSummarizationTask =
                             new ExtractiveSummarizationTask();
                         extractiveSummarizationTask.setParameters(
-                            new ExtractiveSummarizationTaskParameters().setModelVersion(action.getModelVersion()));
+                            new ExtractiveSummarizationTaskParameters()
```

Comment:
yay!!! do you have an issue for adding the properties to the other actions?

Reply:
Yes. I think the first might go to the GA version first, then merge it to latest beta.

Reply:
ahh good point